tooling/metamodel-generator/src/test/java/org/hibernate/jpamodelgen/test/metamodelpackage/MetaModelPackageAnnotationGeneratedTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.jpamodelgen.test.metamodelpackage;
+
+import org.hibernate.jpamodelgen.JPAMetaModelEntityProcessor;
+import org.hibernate.jpamodelgen.test.util.CompilationTest;
+import org.hibernate.jpamodelgen.test.util.TestForIssue;
+import org.hibernate.jpamodelgen.test.util.WithClasses;
+import org.hibernate.jpamodelgen.test.util.WithProcessorOption;
+import org.junit.Test;
+
+import static org.hibernate.jpamodelgen.test.util.TestUtil.assertMetamodelClassGeneratedFor;
+
+/**
+ * @author Marvin S. Addison
+ */
+public class MetaModelPackageAnnotationGeneratedTest extends CompilationTest {
```

Comment:
For completeness could you also test
* Empty string supplied package name
* Not supplying the configuration parameter, therefore assuming default behavior.

Reply:
Pushed commit with additional test cases as requested.